lib/views/issueish-detail-view.js

```diff
@@ -105,6 +107,85 @@ export class BareIssueishDetailView extends React.Component {
     this.refresher.destroy();
   }
 
+  renderIssueBody(issueish, childProps) {
+    return (
+      <Fragment>
+        <GithubDotcomMarkdown
+          html={issueish.bodyHTML || '<em>No description provided.</em>'}
+          switchToIssueish={this.props.switchToIssueish}
+        />
+        {this.renderEmojiReactions(issueish)}
+        <IssueTimelineController
+          {...childProps}
+          switchToIssueish={this.props.switchToIssueish}
+        />
+      </Fragment>
+    );
+  }
+
+  renderPullRequestBody(issueish, childProps) {
+    return (
+      <Tabs>
+        <TabList className="github-IssueishDetailView-tablist">
+          <Tab className="github-IssueishDetailView-tab">
+            <Octicon icon="info" className="github-IssueishDetailView-tab-icon" />Overview</Tab>
+          <Tab className="github-IssueishDetailView-tab">
+            <Octicon icon="checklist" className="github-IssueishDetailView-tab-icon" />
+            Build Status
+          </Tab>
+          <Tab className="github-IssueishDetailView-tab">
+            <Octicon icon="git-commit"
+              className="github-IssueishDetailView-tab-icon"
+            />
+              Commits
+          </Tab>
+        </TabList>
+        {/* 'Files Changed' and 'Reviews' tabs to be added in the future. */}
+
+        {/* overview */}
+        <TabPanel>
+          <GithubDotcomMarkdown
+            html={issueish.bodyHTML || '<em>No description provided.</em>'}
+            switchToIssueish={this.props.switchToIssueish}
+          />
+          {this.renderEmojiReactions(issueish)}
+          <PrTimelineContainer
+            {...childProps}
+            switchToIssueish={this.props.switchToIssueish}
+          /> :
```

Comment:
There is a `:` at the bottom left in the Overview tab:

![screen shot 2018-09-21 at 9 31 58 pm](https://user-images.githubusercontent.com/378023/45881364-edd32b80-bde5-11e8-95ee-e9de57694140.png)

Is it the ☝️ above `:` that could be removed?

Reply:
oh, that's a mistake!  Thanks for pointing it out.  Will remove.